airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -569,16 +578,10 @@ def create(
 class JobsOperations(BaseOperations):
     """Job operations."""
 
-    def list(
-        self, job_type: str, hostname: str, is_alive: bool
-    ) -> JobCollectionResponse | ServerResponseError:
+    def list(self, job_type: str, hostname: str, is_alive: bool) -> list | ServerResponseError:
         """List all jobs."""
-        try:
-            params = {"job_type": job_type, "hostname": hostname, "is_alive": is_alive}
-            self.response = self.client.get("jobs", params=params)  # type: ignore
-            return JobCollectionResponse.model_validate_json(self.response.content)
-        except ServerResponseError as e:
-            raise e
+        params = {"job_type": job_type, "hostname": hostname, "is_alive": is_alive}
+        return super().execute_list(path="jobs", data_model=JobCollectionResponse, params=params,offset = 2,limit = 5)
```

Comment:
Do we need to send offset and limit? Maybe we can even delete this from the method definition if we won't ask the user on the `page` and `limit` :) We can do that enhancement as a next step, let's first not have the ability to set them and we return all